great_expectations/expectations/core/expect_query_results_to_match_source.py

```diff
@@ -180,6 +186,15 @@ def _validate(
                 1 - (match_count / max(source_result_count, target_result_count))
             ) * 100
 
+            missing_rows = self._compute_row_data(
+                col_names=self._get_column_names_from_result(source_results),
+                frequency_map=source_results_frequency_map - target_results_frequency_map,
```

Comment:
If this is the same as using `Counter.subtract()` above, I think that would mean that `source_results_frequency_map` is decremented in-place and therefore it's altered before being used again below.

If that's not the case and using the `-` operator returns something, can you update lines 169-170 to `count_in_source_not_target = source_results_frequency_map - target_results_frequency_map`?

Reply:
`-` actually works totally differently than `.subtract`. It takes it as low as 0, but won't go negative. It feels more like set subtraction (except that you can obviously represent multiple of each element in this data structure). I'll add a comment, and then actually just remove the above lines with the `.subtract` call. Those aren't needed any longer.